lib/lattice_observer/observed/decay.ex

```diff
@@ -6,18 +6,29 @@ defmodule LatticeObserver.Observed.Decay do
   # Based on the decay rate in the lattice parameter, hosts should
   # age out in descending order of health status
   def age_hosts(l = %Lattice{}, event_time) do
+    {fully_decayed, hosts} =
+      age_hosts(l.hosts, event_time, l.parameters.host_status_decay_rate_seconds)
+
+    # Remove each fully decayed host and its resources from the lattice
+    lattice =
+      List.foldl(fully_decayed, l, fn {hk, _host}, lattice ->
```

Comment:
Suggestion: I have a hard time following single-character variable names, and not being very familiar with Elixir's fold syntax, got especially got confused by `l` vs `lattice`

```suggestion
      List.foldl(fully_decayed, lattice, fn {host_key, _host}, acc ->
```